alpha/apps/kaltura/lib/request/infraRequestUtils.class.php

```diff
@@ -455,7 +455,7 @@ public static function getRequestParams()
 			$params[$key['value']] = $value['value'];
 		}
 			
-		self::$requestParams = array_merge($params, $_GET, $_POST, $_FILES);
+		self::$requestParams = array_merge($_FILES, $_POST, $_GET, $params);
```

Comment:
How would you prioritize them against JSON?
See: https://github.com/kaltura/server/pull/2907/files#diff-a40d9041ddca11fe5c490619bdbecf7cR475